src/GitHub.Api/Git/GitClient.cs

```diff
@@ -148,9 +152,39 @@ private Task<NPath> LookForPortableGit()
             return path;
         }
 
-        public bool ValidateGitInstall(NPath path)
+        public ITask<ValidateGitInstallResult> ValidateGitInstall(string path)
         {
-            return path.FileExists();
+            if (!path.ToNPath().FileExists())
+            {
+                return new FuncTask<ValidateGitInstallResult>(cancellationToken, 
+                    () => new ValidateGitInstallResult(false, null, null));
+            }
+
+            Version gitVersion = null;
+            Version gitLfsVersion = null;
+
+            var gitVersionTask = new GitVersionTask(cancellationToken);
+            gitVersionTask.Configure(processManager, path, 
+                gitVersionTask.ProcessArguments, environment.RepositoryPath);
```

Comment:
You can do `var gitVersionTask = new GitVersionTask(cancellationToken).Configure(...)`, Configure returns the task.

Reply:
I needed to get my hands on `gitVersionTask.ProcessArguments` on the second line.

Reply:
Oooh! Just fix the extension method so it passes on the process arguments, in that case, that's what it's for.